src/VerifyDuoController.js

```diff
@@ -111,11 +106,11 @@ function (Okta, Duo, Q, CookieUtil, FactorUtil, FormController, Enums, FormType,
 
       postRender: function () {
         this.add('<iframe frameborder="0"></iframe>');
-        if (this.settings.get('features.rememberDevice')) {
+        if (this.options.appState.get('allowRememberDevice')) {
           this.addInput({
             label: false,
             'label-top': true,
-            placeholder: Okta.loc('rememberDevice', 'login'),
+            placeholder: this.options.appState.get('rememberDeviceLabel'),
             className: 'margin-btm-0',
```

Comment:
`this.appState` instead of `this.options.appState` to keep it consistent with the way we are using it above.